app/panel/components/Header.jsx

```diff
@@ -135,6 +135,12 @@ class Header extends React.Component {
 	}
 
 	clickLogo = () => {
```

Comment:
@Eden12345 can you please rewrite this function using `<Redirect ... />` from `react-router-dom`? It seems like we should have been doing this way all long, instead of messing with the history.

Reply:
I used `<Link ... />` instead of `<Redirect ... />` for two reasons:

1) This routing action is associated with a click event rather than UI state change.

2a) `<Redirect ... />` should only be used when you are redirecting out of a component. This routing action is being triggered from the `<Header />` component, which will not be removed from the DOM when the routing action occurs.

2b) If we wanted to use `<Redirect ... />` here, we would need to handle it in the local state of the `<Panel />` component and pass that state as a prop to the component that is being redirected away from. We would also have to pass a class method for the `<Panel />` component that changes this local state as a prop to the `<Header />` component. This feels more cumbersome than using `<Link ... />`.